rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerClientExample.java

```diff
@@ -49,12 +51,12 @@
     public static void main(String[] args) throws Exception {
 
         // LOGIN IN
-        SchedulerRestClient client = new SchedulerRestClient("http://localhost:9191/rest/rest/");
+        SchedulerRestClient client = new SchedulerRestClient("http://localhost:8080/rest/");
         SchedulerRestInterface scheduler = client.getScheduler();
         String sessionId = scheduler.login("admin", "admin");
 
         // JOB SUBMISSION
-        File xmlJobFile = new File("/home/ybonnaffe/src/cloud_service_provider_conectors/cloudstack/vminfo_job.xml");
+        File xmlJobFile = new File("/path/to/some/existing/job/job_1.xml");
```

Comment:
Not sure to fully understand the intends of this piece of code. Shouldn't be there some kind of checks to see if the file exists ?

Reply:
it is just example of code - we dont run it. It is there so you can use it whenver you want

Reply:
If this is the case, OK from my side